docs/source/conf.py

```diff
@@ -12,11 +12,21 @@
 
 import os
 import sys
+import shutil
 import importlib.util
+import errno
 
 from databricks import koalas
 sys.path.insert(0, os.path.abspath('.'))
 
+# Remove previously generated rst files. Ignore errors just in case it stops generating whole docs.
+shutil.rmtree("%s/reference/api" % os.path.dirname(os.path.abspath(__file__)), ignore_errors=True)
```

Comment:
It's not a big deal but I wanted to avoid to define some variables in case it affects Sphinx unexpectedly.